app/uk/gov/hmrc/hec/util/TimeUtils.scala

```diff
@@ -16,12 +16,13 @@
 
 package uk.gov.hmrc.hec.util
 
-import java.time.{Clock, LocalDate}
+import java.time.{Clock, LocalDate, ZoneId, ZonedDateTime}
 
 object TimeUtils {
 
   val clock: Clock = Clock.systemUTC()
 
-  def today(): LocalDate = LocalDate.now(clock)
+  def today(): LocalDate           = LocalDate.now(clock)
+  def todayByZone(): ZonedDateTime = ZonedDateTime.now(ZoneId.of("Europe/London"))
```

Comment:
maybe a name like `now()` or `nowInEuropeLondon()` is more accurate - `today` doesn't include the concept of time. May not need the `InEuropeLondon` part cos the `ZonedDateTime` will allow users to adjust the date time to whatever zone they need to so the base zone is not necessarily required info

Reply:
so , def nowByZone(): ZonedDateTime = ZonedDateTime.now()? is this you are trying to say? or ZoneId is still required?

Reply:
I meant replacing the name `todayByZone`

Reply:
done